core/src/packet_deserializer.rs

```diff
@@ -55,8 +56,10 @@ impl PacketDeserializer {
 
         // Note: this can be removed after feature `round_compute_unit_price` is activated in
         // mainnet-beta
-        let _working_bank = self.bank_forks.read().unwrap().working_bank();
-        let round_compute_unit_price_enabled = false; // TODO get from working_bank.feature_set
+        let working_bank = self.bank_forks.read().unwrap().working_bank();
+        let round_compute_unit_price_enabled = working_bank
+            .feature_set
+            .is_active(&feature_set::round_compute_unit_price::id());
```

Comment:
For context: banking_stage flags received packets with feature gate status fetched from working_bank, the flag is used to process compute_budget instruction before inserting packet into banking_stage's priority queue.